### DISCUSSION TITLE
Huly

### DISCUSSION THREAD

**Discussion by angelocala94** [coollabsio/coolify#2536] [Service Template Requests]
It’s an open-source everything app that covers Linear, Jira, Slack, and Notion in one platform.

Check it out:
[Huly website](https://huly.io/)
[GitHub](https://github.com/hcengineering/huly-selfhost)

Would love to see it on Coolify!

**Comment by grogou** [comment]
Yes please it is nice tool.

**Comment by SaadBazaz** [comment]
Bumping this up, at least it "looks" nice.

**Comment by AzDarGee** [comment]
Bump

**Comment by Muckdat** [comment]
yes pleaaaaaase

**Comment by YounesAmalou** [comment]
Bumping this

**Comment by mbenitez343** [comment]
bump!
